entry.py

```diff
@@ -0,0 +1,29 @@
+import sys
+    # caution: path[0] is reserved for script path (or '' in REPL)
+sys.path.insert(1, '/Users/gabidoye/Documents/data_Engineering/students_cgpa_python/src')
```

Comment:
Why do you need this?

Reply:
using it to read file from another folder.